plugins/argentina-afip-invoices/src/classes/Argentinacae/exceptionhandler.php

```diff
@@ -4,9 +4,7 @@
 
 set_exception_handler('exception_handler');
 
-function exception_handler(Exception $e) 
+function exception_handler(Exception $e)
```

Comment:
if you're changing all the plugins, you need to update all the plugin manifests and zip archives